cc/crypto/hpke/recipient_context.h

```diff
@@ -19,11 +19,11 @@
 
 #include <memory>
 #include <string>
-#include <tuple>
+#include <vector>
 
 #include "absl/status/statusor.h"
 #include "absl/strings/string_view.h"
-#include "cc/crypto/hpke/utils.h"
+#include "openssl/aead.h"
 #include "openssl/hpke.h"
```

Comment:
Is this still needed?

Reply:
Internal clang-tidy demands to include `aead.h` because of
```
no header providing "EVP_AEAD_CTX_seal" is directly included
```

Reply:
Removed